README.md

```diff
@@ -152,6 +153,28 @@ The storage for Grafana is also mounted as a volume so you can persist any dashb
 
 By using the `--profile tunnel` flag this will start a ngrok tunnel locally that enables traffic forwarding from an arbitrary URL to your local collector. You can configure this forwarding URL and other settings in `tunnel/ngrok.yml` by copying over the example file - `ngrok.yml.example`. In addition you can configure security settings (e.g., only allowing traffic from certain IP ranges / CIDRs) by customising the `ngrok/policy.yml` file. For more configuration options see the ngrok agent config documentation [here](https://ngrok.com/docs/agent/config/v3/).
 
+## Querying data using DuckDB
```

Comment:
Should there be a more generic "Querying data" section that contains both the DuckDB and SQLite subsections?

Reply:
I'm thinking about removing SQLite entirely because I kind of hate the format it writes in as it's a bit too tricky to query for most users (particularly contexts).